PAWI_HS18_Projektdokumentation.tex

```diff
@@ -200,6 +222,13 @@ \subsubsection{Markerless}
 \subsubsection{SLAM}
 Wie im vorherigen Paragraf erklärt, wird zwischen zwei Markerless Tracking Methoden unterscheiden. Eine spezifische, die sogenannte SLAM Methode, wird in diesem Paragraoh näher erklärt. Diese Methode identifiziert, mit Computer Vision, Feature-Points (ein Bildpunkt der sich durch seine lokale detaillierte Umgebung von anderen Punkten desselben Bilds hervorhebt), sowie deren Abhängigkeiten zueinander (Location) und fasst diese in einer Karte zusammen (Mapping). Sollte nun die Pose verloren gehen (durch einen Unterbruch des Kamerastream oder zu schnelle Bewegungen), muss nur ein Teil der vorherigen Karte wiedergefunden werden, um die Position wieder zu bestimmen. Dieser Ansatz wird \textquotedblleft Simultaneous Location and Mapping\textquotedblright (dt. Simultanes Erkennen und Kartografieren) genannt.
 
+\subsubsection{GPS}
```

Comment:
Evt. noch drauf eingehen wie man es für eine AR applikation nun gebrauchen kann?
Wenn nicht drauf eingegegen wird sollte es eine eigene Information werden (eher darauf eingehen kann ja auch nur ein Satz sein)

Reply:
Ich bin der Meinung das müsste dann aber eher in das Kapitel Realisierung, wo wir das GPS gebrauchen um das Problem der Registrierung zu lösen. Deine Meinung?

Reply:
Folgendes, hier befindet sich das Kapitel über AR und dann kommt ein random Text über GPS?
Sehe 3 möglichkeiten dies zu bereinigen.
 1. AR theoretische möglichkeit darlegen da theorie vorhanden ist (einbeziehen was gps mit ar zu tun hat)
 2. GPS als eigene Secrtion (jedoxh fehlt immernoch ein Bezug zu ar)
 3. Textpassage löschen

Reply:
Aber das ist ja genau der Punkt, das Kapitel heisst technologische Grundlagen, nicht technologische Grundlagen von AR
Ich werde aber GPS in eine eigene Section verschieben